infra/terraform/global/main.tf

```diff
@@ -1,8 +1,9 @@
 terraform {
   backend "s3" {
-    bucket = "terraform.analytics.justice.gov.uk"
-    key    = "base/terraform.tfstate"
-    region = "eu-west-1"
+    bucket         = "terraform.analytics.justice.gov.uk"
+    key            = "base/terraform.tfstate"
+    region         = "eu-west-1"
+    dynamodb_table = "terraform-global"
```

Comment:
Doesn't this need to be `terraform-platform-base` to match the table name above?

Reply:
No - there's a DynamoDB table required for each Terraform state object/resource set - `global`, `platform-base` and `platform`. Global TF uses `terraform-global`, which needs to be created manually as it's the very first thing to run, and the global resources then create the two DynamoDB tables for `platform-base` and `platform`.

Reply:
Makes sense. Please do add the creation of the dynamodb table with the other awscli commands in this section: https://github.com/ministryofjustice/analytics-platform-ops#creating-global-aws-resources-and-preparing-terraform-remote-state